### PR TITLE
Pelias

### PR DESCRIPTION
**Pull request by JosseVanDelm**
Add pelias installation documentation for #47. 
Could you guys maybe see if this is clear/ complete?

### CODE REVIEW

File: pelias/README.md [hunk 1]
```diff
@@ -0,0 +1,168 @@
+# Setup guide for Pelias geocoding service.
+
+This guide explains how we set up the geocoding service for https://best.osoc.be.
```
Comment (jbelien) [inline pelias/README.md]: Even though, GitHub make links clickable, try to always put your links between `<>` (like this `<https://best.osoc.be>`) just like you did line 26.
This is only needed when you display the whole URL!

File: pelias/README.md [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+
+This folder contains configuration files to download/prepare/build a complete Pelias installation for Belgium
+using the BOSA open address database and various other open data sources.
```
Comment (jbelien) [inline pelias/README.md]: Add link to BOSA open address website!

File: pelias/README.md [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+It also describes how to set up an apache webserver as a proxy and to make the service serve over [https with certbot](https://certbot.eff.org/).
```
Comment (jbelien) [inline pelias/README.md]: `https with [certbot](https://certbot.eff.org/)` instead !

File: pelias/README.md [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+This guide was adapted from https://github.com/pelias/docker/tree/master/projects/belgium .
+
+# Why Pelias?
+
+We chose pelias for a couple of reasons:
+* It is open source.
+* It offers a very useful API for forward and reverse geocoding, and even for autocompletion.
+* It provides easy [openaddresses.io](https:openaddresses.io) integration. Which makes it particulary interesting for our project.
```
Comment (jbelien) [inline pelias/README.md]: Correct link is `https://openaddresses.io`

**Review by jbelien — CHANGES_REQUESTED** [review]

**Comment by JosseVanDelm** [comment]
Thanks @jbelien ! Will change the stuff!

File: pelias/README.md (hunk 1 above)
Comment (JosseVanDelm) [reply pelias/README.md]: This should be fixed now!

File: pelias/README.md (hunk 3 above)
Comment (JosseVanDelm) [reply pelias/README.md]: Changed

File: pelias/README.md (hunk 2 above)
Comment (JosseVanDelm) [reply pelias/README.md]: Woops forgot this!

Comment (JosseVanDelm) [reply pelias/README.md]: Fixed with 6c4a8a8a30191cb6954ca10a749311e4dfac8e49

File: pelias/README.md (hunk 4 above)
Comment (JosseVanDelm) [reply pelias/README.md]: Fixed as well!

**Review by jbelien — APPROVED** [review]

**Review by theodedeken — APPROVED** [review]

**Comment by JosseVanDelm** [comment]
Thank you guys!

**Pull request merged**